### ISSUE TITLE
free software license?

### ISSUE THREAD

**Issue by foundscapes** [OpenArchive/OpenArchive#10]
We need to make sure the app. is licensed this way.

**Comment by vitriolix** [comment]
The app is GPL

**Comment by mattl** [comment]
I'll do a spot check on all the files before we release, to make sure we're compliant.

**Comment by vitriolix** [comment]
Actually I believe we do need to add them to the file headers still, we only have a LICENCSE

**Comment by mattl** [comment]
Yeah, I'll take on this fun task and ping if you if I have any questions
